packages/nimble-components/src/rich-text-editor/tests/rich-text-editor-matrix.stories.ts

```diff
@@ -23,13 +26,32 @@ const metadata: Meta = {
     }
 };
 
+const richTextMarkdownString = '1. **Bold*Italics***';
+
 export default metadata;
 
 // prettier-ignore
 const component = (): ViewTemplate => html`
     <${richTextEditorTag}></${richTextEditorTag}>
 `;
 
+const [
+    lightThemeWhiteBackground,
+    colorThemeDarkGreenBackground,
+    darkThemeBlackBackground,
+    ...remaining
+] = backgroundStates;
+
+if (remaining.length > 0) {
+    throw new Error('New backgrounds need to be supported');
+}
```

Comment:
If you replace uses of `createFixedThemeStory` with `createMatrixThemeStory` and `createStory` as I suggest in a couple other comments, you can remove this.

Reply:
Updated as per your suggestion.